source/extensions/filters/http/jwt_authn/authenticator.h

```diff
@@ -42,7 +42,8 @@ class Authenticator {
   // Authenticator factory function.
   static AuthenticatorPtr create(const ::google::jwt_verify::CheckAudience* check_audience,
                                  const absl::optional<std::string>& provider, bool allow_failed,
-                                 JwksCache& jwks_cache, Upstream::ClusterManager& cluster_manager);
+                                 JwksCache& jwks_cache, Upstream::ClusterManager& cluster_manager,
```

Comment:
do we still need cluster_manager?   I through it is only needed for JwksFetcher.